slides.md

```diff
@@ -552,7 +552,89 @@ http://localhost:3000/login
 
 ---
 
-# Step 5: Testing
+# Step 5 Constraints
+
+- Route validation can also be constrained to match properties of the request. By default fastify supports `version` (via `Accept-Version` header) and `host` (via `Host` header)
+
+> 🏆 Custom constraints can be added via [`find-my-way`](https://github.com/delvedor/find-my-way)
+
+https://www.fastify.io/docs/latest/Reference/Routes/#constraints
+
+---
+
+# Step 5: Exercise 💻
+
+<div class="dense">
+
+- Add a new `GET /version` route that only accepts requests matching version `1.0.0`
+
+> 💡 The `Accept-Version` header should accept 1.x, 1.0.x and 1.0.0
+</br>
```

Comment:
This line break is breaking the slide presentation. It could be substituted by and empty line to achieve the spacing.

Reply:
True, that should be fixed now